File: app/src/main/java/com/androidacademy/msk/exerciseproject/screen/intro/PageFragment.java
package com.androidacademy.msk.exerciseproject.screen.intro;

import android.os.Bundle;
import android.support.annotation.DrawableRes;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ImageView;

import com.androidacademy.msk.exerciseproject.R;

public class PageFragment extends Fragment {

    private static final String KEY_IMAGE = "KEY_IMAGE";

    public static PageFragment newInstance(@DrawableRes int drawableRes) {

        Bundle args = new Bundle();
        args.putInt(KEY_IMAGE, drawableRes);

        PageFragment fragment = new PageFragment();
        fragment.setArguments(args);
        return fragment;
    }

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment_page, container, false);

        int drawableRes = getArguments().getInt(KEY_IMAGE);

        ImageView imageView = view.findViewById(R.id.imageview_page);
        imageView.setImageResource(drawableRes);


        return view;
    }
}

File: app/src/main/java/com/androidacademy/msk/exerciseproject/screen/intro/PageFragment.kt
package com.androidacademy.msk.exerciseproject.screen.intro

import android.os.Bundle
import android.support.annotation.DrawableRes
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.androidacademy.msk.exerciseproject.R
import kotlinx.android.synthetic.main.fragment_page.*

class PageFragment : Fragment() {

    companion object {

        private const val KEY_IMAGE = "KEY_IMAGE"

        fun newInstance(@DrawableRes drawableRes: Int): PageFragment {
            val args = Bundle()
            args.putInt(KEY_IMAGE, drawableRes)

            val fragment = PageFragment()
            fragment.arguments = args
            return fragment
        }
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_page, container, false)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        arguments?.getInt(KEY_IMAGE)?.apply { imageview_page.setImageResource(this) }
    }
}
